src/debugAdapter/goDebug.ts

```diff
@@ -201,6 +201,7 @@ interface DiscardedBreakpoint {
 
 // This interface should always match the schema found in `package.json`.
 interface LaunchRequestArguments extends DebugProtocol.LaunchRequestArguments {
+	kind: 'launch';
```

Comment:
Do we need an extra `kind` field? Can we just use the `request` field?

Reply:
Good point, fixed.